Bottom: 911b92c37714f963c09759c1a1b378bcb15d0f95
Top:    ca645550e0050e1492659e779400701705bad29e
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-04-16 16:08:07 +0800

Refresh of vhost-net-tx-batching

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index a4d7ec3..e97ede5 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -472,7 +472,7 @@ static int batch_tx(struct vhost_net *net,
 		return err;
 	}
 	vhost_add_used_and_signal_n(&net->dev, vq, nvq->heads, n);
-	vhost_net_tx_packet(net, VHOST_RX_BATCH);
+	vhost_net_tx_packet(net, n);
 
 	return 0;
 }
